web/jenkins.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Jenkins slave
+
+[Service]
+User=omero
+WorkingDirectory=/tmp
+PIDFile=/tmp/jenkins.pid
+Type=forking
+Restart=on-failure
+RestartSec=10
+ExecStart=/tmp/jenkins-slave.sh
```

Comment:
you mean https://documentation.cloudbees.com/docs/dev-at-cloud/On-Premise+Executors.html?query=systemd#running-the-slave-with-systemd-on-linux
???

```
[root@102787622c66 ~]# systemctl status jenkins.service
● jenkins.service - Jenkins slave
   Loaded: loaded (/etc/systemd/system/jenkins.service; enabled; vendor preset: disabled)
   Active: active (running) since Fri 2016-03-25 16:04:12 UTC; 1min 14s ago
     Docs: https://developer.cloudbees.com/bin/view/DEV/On-Premise+Executors
 Main PID: 216 (java)
   CGroup: /system.slice/jenkins.service
           └─216 java -jar /tmp/swarm-client-2.0-jar-with-dependencies.jar -fsroot /home/omero -labels centos7 -labels java -name web -disableClientsUniq...

Mar 25 16:05:03 102787622c66 jenkins-slave.sh[216]: Retrying in 10 seconds
Mar 25 16:05:16 102787622c66 systemd[1]: Started Jenkins slave.
Mar 25 16:05:18 102787622c66 jenkins-slave.sh[216]: Failed to receive a reply to broadcast.
```